src/librustc/mir/interpret/error.rs

```diff
@@ -175,6 +176,7 @@ pub enum EvalErrorKind<'tcx, O> {
     NoMirFor(String),
     UnterminatedCString(Pointer),
     DanglingPointerDeref,
+    ExternStaticRead(DefId),
```

Comment:
This is introduced here and removed two commits later. Merge the commits into each other?